### PR TITLE
docs(contribution): describe new documentation

### PR DESCRIPTION
**Pull request by JWO719**
This pr adds some minor informations about the new documentation to the contribution.md.

### CODE REVIEW

**Comment by moniuch** [comment]
Hi @JWO719,

I am just going through my PR #3774 where I was a bit unsure about the image links to the operators' marble diagrams, especially the `do/tap` operator. The link in `src/internal/operators/tap.ts` still refers to the old `do` - are those marble images generated automatically? Shouldn't we perhaps have have a short info about handling/updating marbles in the docs? What do you think :)

**Comment by JWO719** [comment]
First of all thanks for the hint, you are totally right. Actually I am not aware how/who generates/design the images. Maybe @benlesh knows more about that. @moniuch maybe create an issue for the tap/do image

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -122,9 +122,12 @@ from the main (upstream) repository:
 
 ## Documentation
 
-- Documentation of RxJs will be shortly inline and more detailed in [rxjs-docs](https://github.com/ReactiveX/rxjs-docs)
-- The [Documentation Guidelines](https://github.com/ReactiveX/rxjs-docs/blob/master/DOCUMENTATION_GUIDELINES.md) will support you
-- Please keep both documents up to date
+- The documentation is auto-generated directly from the source code.
+- In short: From the source code we generate JSON documents, describing each operator, function ... and render this JSON within an Angular application
+- The folder `docs-app` contains everything you need for building and developing the docs
+- the folder `doc` used to be the documentation, but should remain until all content is transfered.
```
Comment (benlesh) [inline CONTRIBUTING.md]: This isn't *quite* accurate. That folder contains additional documentation that we were amending to the old generated documentation.

Comment (JWO719) [reply CONTRIBUTING.md]: Until today I thought we would update the old docs by using dgeni too, but I don't know. Never thought about that :D I don't mind changing it

**Comment by sumitarora** [comment]
@JWO719 👍

**Comment by benlesh** [comment]
Thanks @JWO719!

**Pull request merged**